### PR TITLE
fix PEP8 issues in classes without .py extension and Test script resulting in errors failing Travis. Fix JSON Output

### PR DESCRIPTION
**Pull request by dITySoftware**
fix PEP8
fix JSON Output

### CODE REVIEW

File: aecid-testsuite/runCoverageTests.sh [hunk 1]
```diff
@@ -26,3 +28,4 @@ kafka_2.12-2.5.0/bin/zookeeper-server-stop.sh > /dev/null
 sudo rm -r kafka_2.12-2.5.0/
 sudo rm -r /tmp/zookeeper
 sudo rm -r /tmp/kafka-logs
+exit `expr "$exit_code1" + "$exit_code2"`
```
Comment (landauermax) [inline aecid-testsuite/runCoverageTests.sh:31]: I am not sure if this is good coding style. Maybe it is better to have an if statement that checks if each exit code is zero, and returns 0 when true and 1 otherwise.

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/EnhancedNewMatchPathValueComboDetector.py [hunk 2]
```diff
@@ -1,17 +1,6 @@
...
-This program is free software: you can redistribute it and/or modify it under
-the terms of the GNU General Public License as published by the Free Software
-Foundation, either version 3 of the License, or (at your option) any later
-version.
-This program is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
-FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
-You should have received a copy of the GNU General Public License along with
-this program. If not, see <http://www.gnu.org/licenses/>.
-"""
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/EnhancedNewMatchPathValueComboDetector.py:14]: Why was this deleted? It is in the header of all files.

Comment (dITySoftware) [reply source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/EnhancedNewMatchPathValueComboDetector.py]: This was deleted by mistake. I readded it.

File: aecid-testsuite/runCoverageTests.sh (hunk 1 above)
Comment (dITySoftware) [reply aecid-testsuite/runCoverageTests.sh:31]: Coverage tests usually return only 0 and 1. As this script is running two Coverage tests the summed exit code would show if
- 0 both tests were successful
- 1 one test failed
- 2 both tests failed

Otherwise the success / failure could be seen in the output of the unittests. That just were my thoughts on it.

Should i change or leave it?
